### PR TITLE
Build UI Detail Page

### PR DESCRIPTION
**Pull request by raymond-le-goldenowl**
**Task**

- Detail page

**Descriptions**
- Base skeleton detail page

**Screenshots**
Screenshot from 2021-12-30 12-21-27

### CODE REVIEW

File: src/pages/home-page.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      </h1>
+      {/* ! Search Form*/}
+      <form id="form-search" className="mb-5">
+        <label htmlFor="SearchPokemon" className="form-label">
+          Search pokemon
+        </label>
+        <input
+          type="text"
+          className="form-control"
+          id="SearchPokemon"
```
Comment (jun-nguyen-goldenowl) [inline src/pages/home-page.js]: `id` should be in hyphen-separated case `search-pokemon-input`

File: src/pages/home-page.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+export default function HomePage() {
+  return (
+    <div className="container-fluid">
+      {/* Logo */}
+      <h1 className="text-center mb-5" id="logo-page">
+        Pokemon Dex
+      </h1>
+      {/* ! Search Form*/}
+      <form id="form-search" className="mb-5">
+        <label htmlFor="SearchPokemon" className="form-label">
```
Comment (jun-nguyen-goldenowl) [inline src/pages/home-page.js]: id should be in hyphen-separated case `search-pokemon-input`

**Review by jun-nguyen-goldenowl — CHANGES_REQUESTED** [review]

File: src/pages/home-page.js (hunk 1 above)
Comment (raymond-le-goldenowl) [reply src/pages/home-page.js]: fixed Issue

File: src/pages/home-page.js (hunk 2 above)
Comment (raymond-le-goldenowl) [reply src/pages/home-page.js]: fixed Issue

**Review by jun-nguyen-goldenowl — APPROVED** [review]

**Pull request merged**